web_src/js/features/contributors.js

```diff
@@ -0,0 +1,28 @@
+import {createApp} from 'vue';
+
+export async function initRepoContributors() {
+  const el = document.getElementById('repo-contributors-chart');
+  if (!el) return;
+
+  const {default: RepoContributors} = await import('../components/RepoContributors.vue');
```

Comment:
```suggestion
  const RepoContributors = await import('../components/RepoContributors.vue');
```
should be the same as far as I know.

Reply:
It's not the same. I don't know the rules behind this but if it works with `default`, it should not be changed.